src/type.cc

```diff
@@ -281,5 +281,81 @@ GITypeTag GetStorageType (GITypeInfo *type_info) {
     return type_tag;
 }
 
+bool IsTypeSimple (GITypeInfo *type_info) {
```

Comment:
We are not using this function anywhere, do we?
Maybe it should be added later when it's really needed.

Reply:
No we're not. I just feel safer with it around. It's such a nice function. But yes I gess I'll remove it.